### ISSUE TITLE
[Feature Request] Support for URL based rules

### ISSUE THREAD

**Issue by tddschn** [darkreader/darkreader#4198]
**Is your feature request related to a problem? Please describe.**
I want `google.com` in dark mode but `google.com/maps` in the original color, but since they have the same domain `google.com`, I can't seem to find a way to make it work in Google Chrome.

**Describe the solution you'd like**
Add a button "Toggle Current URL" that allows users to edit the current URL to a proper one like `*://google.com/maps/*` and add a rule for this URL.

**Additional context**
Google search result pages generally look good in dark mode, but I cannot use Google Maps with ease in dark mode. :(

**Comment by Gusted** [comment]
Hi @tddschn!

> Add a button "Toggle Current URL" that allows users to edit the current URL to a proper one like *://google.com/maps/* and add a rule for this URL.

What benefits have this over going to the sitelist and adding `*://google.com/maps/*` to the list and having an Toggle Current URL which is editable.

Regards,
Gusted

**Comment by tddschn** [comment]
@Gusted You are right :) Thr dark reader team can implement this feature however they see fit.

Regards,
Teddy
